### ISSUE TITLE
Difference in sorting between original DP and the wizard

### ISSUE THREAD

**Issue by jsonnentag** [rrirower/DP2SceneGraphWizard#99]
I can see doing it either way, but kind of see the reason why the original DP sorted the list of Series by the dates of the actual episodes and not just the releaseDate listed for the entire Series.

So the original DP when sorting by the latest listed first would place the Series that had the newest episodes at the front of the list, but I noticed while testing Beta 5 that things weren't appearing where I expected for the Series.  (It doesn't matter for the individual episodes, since they only have 1 releaseDate associated with them.)  I was expecting to see the series with the most recently added episodes (with the most recent episode releaseDate) shown first, but it was down near the end because the original start of the series is in 2020.  :)

Again, I can understand doing the sorting either way, but I just got used to the convenience of the series with active newly added episodes being listed first.  I could update the Series releaseDate each time a new episode is listed and get the same original effect, but if you were aiming for replication of how Roku originally had things working it used episodes in the series rather than the actual listed series date to determine ordering of things (at least when sorting for latest series to be displayed first - I don't know how it worked if you wanted the oldest first, whether they would list Series by the "oldest newest episode" or what).

**Comment by rrirower** [comment]
The sorting algorithm uses the sort order (most_recent or chronological) to sort the items in the Grid screen.  The algorithm does not affect the Episodes in a Series or Season.  This has not yet been implemented.

**Comment by jsonnentag** [comment]
> The sorting algorithm uses the sort order (most_recent or chronological) to sort the items in the Grid screen. The algorithm does not affect the Episodes in a Series or Season. This has not yet been implemented.

Yes, that is the same sorting I am talking about.  But that sorting on the Grid screen (main screen with categories) is not the same as original DP.
